jobs/ci-kubernetes-node-kubelet-dockerized.sh

```diff
@@ -22,8 +22,8 @@ set -o xtrace
 
 readonly testinfra="$(dirname "${0}")/.."
 
-export NODE_SCRIPT="test/e2e_node/jenkins/e2e-node-jenkins.sh"
-export NODE_PROPERTIES="test/e2e_node/jenkins/jenkins-ci.properties"
+export NODE_TEST_SCRIPT="./test/e2e_node/jenkins/e2e-node-jenkins.sh"
```

Comment:
yeah, the `WORKDIR` is set to `/workspace` in kubekins-test, so that's probably the directory we're running in, but the kubernetes repo is mapped to `/go/src/k8s.io/kubernetes`. Maybe `cd` there first?